### PR TITLE
Make the McLunky Lunk

### PR DESCRIPTION
**Pull request by ZombieTsar**
Adds a new generator that can be created from any standard autolathe.

It's wood powered, and at minimal risk of dismemberment on your behalf.

Baseline starts out with a max of 120kw

### CODE REVIEW

**Comment by ZombieTsar** [comment]
I also added the pickaxe to the autolathe, because it wasn't

File: sandcode/code/modules/research/designs/autolathe_designs/autolathe_designs_electronics.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/datum/design/airalarm_electronics
```
Comment (SandPoot) [inline sandcode/code/modules/research/designs/autolathe_designs/autolathe_designs_electronics.dm:1]: can you tell me why is this like this?

File: sandcode/code/modules/research/designs/autolathe_designs/autolathe_designs_electronics.dm [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/datum/design/airalarm_electronics
+	name = "McLunky Circuit Board"
+	id = "mclunky_circuit_board"
+	build_type = AUTOLATHE | PROTOLATHE
+	materials = list(/datum/material/iron = 1000, /datum/material/glass = 2000)
+	build_path = /obj/item/circuitboard/machine/pacman/mclunky
+	category = list("initial", "Electronics")
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
```
Comment (SandPoot) [inline sandcode/code/modules/research/designs/autolathe_designs/autolathe_designs_electronics.dm:8]: add it to service

Comment (SandPoot) [reply sandcode/code/modules/research/designs/autolathe_designs/autolathe_designs_electronics.dm]: actually don't, they won't even the pieces for it anyways

**Pull request merged**